parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -692,7 +752,7 @@ pub mod pallet {
 			let index_in_array = block_slot % max_slots_per_historical_root;
 			let leaf_index = max_slots_per_historical_root + index_in_array;
 
-			log::info!(
+			log::debug!(
 				target: "ethereum-beacon-client",
 				"💫 Depth: {} leaf_index: {}", config::BLOCK_ROOT_AT_INDEX_PROOF_DEPTH, leaf_index
 			);
```

Comment:
We can remove this, I think I missed removing this while testing the ancestry proofs. 😄 

```suggestion
```